### DISCUSSION TITLE
Bot Blocking

### DISCUSSION THREAD

**Discussion by ernmander** [mastodon/mastodon#16743] [General]
Is there or will there be a way of universally blocking bot accounts? The federated view tends to be full of bots spewing out random rubbish. Clogging up the timeline making it harder to find real people of interest to follow.

**Comment by shleeable** [comment]
I would LOVE to be able to hide bots in timelines.

**Reply by leolux** [reply]
Why do bots exist n social networks anyway? Bots are no humans thus not social!
I would also support the development of an bot-detection mechanism. There are tools like signed tokens, browser integrity check and DID from "Web of Trust" which could solve this problem in the long tearm.
